<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CLD-LAB0-QAA01" name="com.whs.navigator.testcases.chamber.SSOLoginTestCase_Sukur" tests="2" failures="0" timestamp="3 Mar 2017 09:18:37 GMT" time="245.050" errors="1">
  <testcase name="validateSetPasswordFromLHP" time="102.190" classname="com.whs.navigator.testcases.chamber.SSOLoginTestCase_Sukur"/>
  <testcase name="verifySSODirectsUserToDfltPortal" time="142.860" classname="com.whs.navigator.testcases.chamber.SSOLoginTestCase_Sukur">
    <error type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.whs.navigator.commonUtil.WebObjectUtil.stopCurrentTestExecution(WebObjectUtil.java:1821)
	at com.whs.navigator.commonUtil.WebObjectUtil.SetValueEdit(WebObjectUtil.java:458)
	at com.whs.navigator.testcases.LoginTestCase.nav_Login_legacy(LoginTestCase.java:226)
	at com.whs.navigator.testcases.LoginTestCase.nav_Login(LoginTestCase.java:1726)
	at com.whs.navigator.testcases.LoginTestCase.LoginAsQaautoAUser(LoginTestCase.java:1383)
	at com.whs.navigator.testcases.chamber.SSOLoginTestCase_Sukur.verifySSODirectsUserToDfltPortal(SSOLoginTestCase_Sukur.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- verifySSODirectsUserToDfltPortal -->
</testsuite> <!-- com.whs.navigator.testcases.chamber.SSOLoginTestCase_Sukur -->
